src/components/FeatureSetsPanel/FeatureSetsPanelTargetStore/featureSetsPanelTargetStore.util.js

```diff
@@ -155,7 +155,9 @@ export const targetsPathEditDataInitialState = {
  */
 export const generatePath = (prefixes, project, kind, name, suffix) => {
   if (prefixes) {
-    const path = prefixes[kind] || prefixes.default
+    let path = prefixes[kind] || prefixes.default
+
+    path = kind === REDISNOSQL ? path.replace('//', '//{hostIP}:{port}') : path
```

Comment:
don't use another replace method, you can add '//' to replace function that we use a few lines below